publish.py

```diff
@@ -1143,7 +1143,10 @@ def package_ui(self):
 
             with zipfile.ZipFile(zipfile_path, "w", zipfile.ZIP_DEFLATED) as zipf:
                 ui_dir = "src/ui"
+                exclude_dirs = {"node_modules", "build"}
                 for root, dirs, files in os.walk(ui_dir):
+                    # Exclude specified directories from zipping
+                    dirs[:] = [d for d in dirs if d not in exclude_dirs]
```

Comment:
interesting - this looks like you fixed a separate bug unrelated to s3 vectors..   Were we zipping up node_modules? Ouch.